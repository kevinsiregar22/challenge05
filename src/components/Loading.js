import {View, ActivityIndicator} from 'react-native';
import React from 'react';
import {ms} from 'react-native-size-matters';

export default function Loading() {
  return (
    <View flex={1} justifyContent="center" alignItems="center">
      <ActivityIndicator size={ms(120)} />
    </View>
  );
}
