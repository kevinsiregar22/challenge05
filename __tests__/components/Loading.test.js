import React from 'react';
import {render} from '@testing-library/react-native';
import Loading from '../../src/components/Loading';

describe('should loading', () => {
  test('should first loading', () => {
    const component = render(<Loading />);
    expect(component).toMatchSnapshot();
  });
});
