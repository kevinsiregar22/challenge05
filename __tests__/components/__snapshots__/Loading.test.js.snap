// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should loading should first loading 1`] = `
<View
  alignItems="center"
  flex={1}
  justifyContent="center"
>
  <ActivityIndicator
    size={188.57142857142856}
  />
</View>
`;
